lms/templates/sysadmin_dashboard_gitlogs.html

```diff
@@ -102,64 +137,69 @@ <h2 class="instructor-nav">
-                  %endif
-                  ${cil.git_log}
-                </td>
+                <th width="15%">${_('Date')}</th>
+                <th width="15%">${_('Course ID')}</th>
+                ## Translators: Git is a version-control system; see http://git-scm.com/about
+                <th>${_('Git Action')}</th>
               </tr>
-
-              ## Show the full log of the latest import if viewing logs for a specific course
-              %if course_id is not None:
-                  <tr class="import-log" id="import-log-${index}">
-                    <td colspan="3">
-                      <pre>
-                        ${cil.import_log | h}
-                      </pre>
+            </thead>
+            <tbody>
+              %for index, cil in enumerate(logs):
+                  <%
+                      # Appropriate datetime string for current locale and timezone
+                      date = get_time_display(cil.created.replace(tzinfo=UTC),
+                                              DEFAULT_DATE_TIME_FORMAT, coerce_tz=settings.TIME_ZONE)
+                  %>
+                  <tr>
+                    <td>${date}</td>
+                    <td>
+                      <a href="${reverse('gitlogs_detail', kwargs={'course_id': unicode(cil.course_id)})}">
+                        ${cil.course_id | h}
+                      </a>
+                    </td>
+                    <td>
+                      %if course_id is not None:
+                          <a class="toggle-import-log" data-import-log="${index}" href="#">[ + ]</a>
+                      %endif
+                      ${cil.git_log}
                     </td>
                   </tr>
-              %endif
-          %endfor
-        </tbody>
-      </table>
-
-      ## If viewing a single course and there are no logs available, let the user know
-      %if course_id is not None and cil is None:
-        ## Translators: git is a version-control system; see http://git-scm.com/about
-        ${_('No git import logs have been recorded for this course.')}
+
+                  ## Show the full log of the latest import if viewing logs for a specific course
+                  %if course_id is not None:
+                      <tr class="import-log" id="import-log-${index}">
+                        <td colspan="3">
+                          <pre>
+                            ${cil.import_log | h}
+                          </pre>
+                        </td>
+                      </tr>
+                  %endif
+              %endfor
+            </tbody>
+          </table>
+
+          ${pagination()}
+      %else:
+          <div class="page-status">
+            %if not course_id:
+                # If viewing all logs there are no logs available, let the user know
+                ## Translators: git is a version-control system; see http://git-scm.com/about
+                ${_('No Git import logs have been recorded.')}
```

Comment:
Is it worthwhile to capitalize "git" here? This creates a new translation string and causes additional translation work, simply due to the change in capitalization (there is no notion of changed strings, only new strings). So if it's not critically necessary to change the string I would advise against it.